resource/js/tab-alpha.js

```diff
@@ -130,13 +177,13 @@ tabAlphaApp.component('tab-alpha', {
       </ul>
     </template>
     
-    <template v-if="loadingConcepts">
-      <div>
-        Loading...
-      </div>
-    </template>
-    <template v-else>
-      <div class="sidebar-list" :style="getListStyle()">
+    <div class="sidebar-list" :style="getListStyle()" ref="list">
+      <template v-if="loadingConcepts">
+        <div>
+          Loading...
```

Comment:
This could be a value from the window.SKOSMOS object, e.g. "Ladataan sisältöä", "Laddar mera resultat" & "Loading more items". Also if you want to use a spinner, something like `<i class="fa-solid fa-spinner fa-spin-pulse"></i>` could be considered.

Reply:
Not sure if it's worth adding more translations to the SKOSMOS object, because the translation mechanism for the frontend may still change to something completely different that doesn't rely on the SKOSMOS object (e.g. reading the translations directly from a JSON file). See #1525 and https://github.com/NatLibFi/Skosmos/issues/1523#issuecomment-1736810725 for some thoughts